content/post/what-is-nodejs-loaders.mdx

```diff
@@ -1,10 +1,26 @@
 ---
 title: What is Nodejs-Loaders
-description: Nodejs-Loaders is a collection of loaders for Node.js.
+description: nodejs-loaders is a project designed to provide a variety of loaders that make local development and CI testing quicker and easier.
 authors: AugustinMauroy, JakobJingleheimer
 date: 2025-07-01
 category: information
 ---
 
+## Our Values: Simplicity, Lightness, Efficiency
+
+We believe in keeping things simple and efficient. nodejs-loaders is built with these core values in mind:
+
+* **Simplicity:** Easy to use and understand.
+* **Lightness:** Minimal overhead and dependencies.
+* **Efficiency:** Optimized for speed and performance.
+
+## Maintained By
+
+nodejs-loaders is currently maintained by [Jacob Smith](https://github.com/JakobJingleheimer) and [Augustin Mauroy](https://github.com/AugustinMauroy/).
```

Comment:
I would use our screen-names since that's what people will probably recognise.

```suggestion
nodejs-loaders is currently maintained by [@JakobJingleheimer](https://github.com/JakobJingleheimer) and [@AugustinMauroy](https://github.com/AugustinMauroy/).
```